subgraphs/compound-forks/protocols/moonwell/src/constants.ts

```diff
@@ -45,6 +45,18 @@ export function getProtocolData(): NetworkSpecificConstant {
     );
   }
 
+  if (equalsIgnoreCase(network, Network.BASE)) {
+    return new NetworkSpecificConstant(
+      Address.fromString("0xfBb21d0380beE3312B33c4353c8936a0F13EF26C"),
```

Comment:
I know it doesn't affect it bc we are turning these strings into addresses right away, but let's update to lowercase for best practice.

I have this new vs code extension you might want to try: https://marketplace.visualstudio.com/items?itemName=ruiquelhas.vscode-lowercase